e2e/release/src/circular-dependencies.test.ts

```diff
@@ -701,11 +701,6 @@ describe('nx release circular dependencies', () => {
 
         Would stage files in git with the following command, but --dry-run was set:
         git add {project-name}/package.json
-
-        NX   Your filter "{project-name}" matched the following projects:
```

Comment:
This a solid improvement, previously the `nx release` "meta command" would end up repeating the same project filters because each subcommand was logging it directly as well